content/chainguard/libraries/python/build-configuration.md

```diff
@@ -193,6 +193,13 @@ Example for `~/.pip/pip.conf`:
 index-url = https://CG_PULLTOKEN_USERNAME:CG_PULLTOKEN_PASSWORD@libraries.cgr.dev/python/simple/
 ```
 
+Note that `pip` does not support installing Python libraries from multiple 
+indexes while prioritizing one index over another. If you are using `pip` 
+and prefer to pull from multiple indexes while prioritizing the Chainguard 
```

Comment:
I think we should stay consistent in the naming - Chainguard Libraries for Python is used everywhere else in the docs